generate_ai_dimensions.py の必須引数不足エラー

GitHub Actions の `manual-workflows.yml` ワークフローで `src/generate_ai_dimensions.py` を呼び出す際に、必須引数である `type` または `--prompt_file` が指定されていません。

**エラーメッセージ:**
```
positional arguments:
  {selia,lyra}          事前定義済みの次元タイプを選択します (selia or
                        lyra)。--prompt_fileとは併用できません。

options:
  -h, --help            show this help message and exit
  --prompt_file PROMPT_FILE
                        使用するカスタムプロンプトのファイルパス。
  --output OUTPUT       出力先のJSONファイルパス。--prompt_file使用時は必須です。
Error: Process completed with exit code 1.
```

**発生箇所:**
`.github/workflows/manual-workflows.yml` の79行目

**解決策の提案:**
`manual-workflows.yml` 内の `generate_ai_dimensions.py` の呼び出しを修正し、以下のいずれかの方法で引数を指定してください。

1.  **事前定義済みタイプを使用する場合:**
    `python src/generate_ai_dimensions.py selia` (または `lyra`)

2.  **カスタムプロンプトを使用する場合:**
    `python src/generate_ai_dimensions.py --prompt_file path/to/your_prompt.txt --output path/to/your_output.json`

--- a/.github/workflows/manual-workflows.yml
+++ b/.github/workflows/manual-workflows.yml
@@ -76,7 +76,7 @@ jobs:
     - name: Generate AI image vectors
       run: python src/generate_ai_image_vectors.py --single_image_path sigma_images/circle_center.jpg
     - name: Generate AI dimensions
-      run: python src/generate_ai_dimensions.py
+      run: python src/generate_ai_dimensions.py selia
     - name: Run terrier comparison script
       run: python scripts/run_terrier_comparison.py
     - name: Run psyche simulation script

